test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1928,57 +1936,25 @@ def forward(self, x):
 
         m1 = TestHelperModules.ConvWithBNRelu(relu=False, bias=False)
         example_inputs = (torch.randn(3, 3, 5, 5),)
-        self._verify_symmetric_qnnpack_qat_graph(
-            m1, example_inputs, is_per_channel=False, has_relu=False, has_bias=False,
-        )
-        m1 = TestHelperModules.ConvWithBNRelu(relu=False, bias=False)
-        self._verify_symmetric_qnnpack_qat_graph(
-            m1, example_inputs, is_per_channel=True, has_relu=False, has_bias=False,
-        )
-        m1 = TestHelperModules.ConvWithBNRelu(relu=False, bias=False)
-        self._verify_symmetric_qnnpack_qat_numerics(
-            m1, example_inputs, is_per_channel=False, verify_convert=True,
-        )
-        m1 = TestHelperModules.ConvWithBNRelu(relu=False, bias=False)
-        self._verify_symmetric_qnnpack_qat_numerics(
-            m1, example_inputs, is_per_channel=True, verify_convert=True,
-        )
-        self._verify_symmetric_qnnpack_qat_numerics(
-            M2(), example_inputs, is_per_channel=False, verify_convert=True,
-        )
-        self._verify_symmetric_qnnpack_qat_numerics(
-            M2(), example_inputs, is_per_channel=True, verify_convert=True,
+        self._verify_symmetric_xnnpack_qat_graph(
+            m1, example_inputs, has_relu=False, has_bias=False,
         )
+        self._verify_symmetric_xnnpack_qat_numerics(m1, example_inputs)
+        self._verify_symmetric_xnnpack_qat_numerics(M2(), example_inputs)
```

Comment:
should this be in a different test?

Reply:
Maybe, but I'm not sure if I want to create a test just for this case so maybe I'll just leave this here for now